libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -710,13 +710,6 @@ void AC_PosControl::update_xy_controller()
     if (!limit_accel_xy(_vel_desired.xy(), _accel_target.xy(), accel_max)) {
         // _accel_target was not limited so we can zero the xy limit vector
         _limit_vector.xy().zero();
```

Comment:
If I understand the bug correctly it is the `_fwd_pitch_is_limited` is not set in this code path. A alternate fix would be to add `_fwd_pitch_is_limited = false;`  here?

Reply:
Leaving this in copter would not be a good thing in my humble opinion. It has nothing to do with copter.

Reply:
Right, I agree with the move, just trying to understand the bugfix.